File: openapi/code_samples/java/api_v1_logs_authentication_requests/get.java
// List authentication request logs (most recent first)
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create(System.getenv("SCALEKIT_ENVIRONMENT_URL")
        + "/api/v1/logs/authentication/requests?page_size=10&status=SUCCESS"))
    .header("Authorization", "Bearer " + accessToken)
    .GET()
    .build();

HttpResponse<String> response = HttpClient.newHttpClient()
    .send(request, HttpResponse.BodyHandlers.ofString());
// response.body() contains authRequests, next_page_token, total_size

File: openapi/code_samples/go/api_v1_logs_authentication_requests/get.go
package main

import (
	"fmt"
	"io"
	"net/http"
	"os"
)

// List authentication request logs (most recent first)
func listAuthRequests(accessToken string) error {
	url := fmt.Sprintf(
		"%s/api/v1/logs/authentication/requests?page_size=10&status=SUCCESS",
		os.Getenv("SCALEKIT_ENVIRONMENT_URL"),
	)
	req, err := http.NewRequest(http.MethodGet, url, nil)
	if err != nil {
		return err
	}
	req.Header.Set("Authorization", "Bearer "+accessToken)

	res, err := http.DefaultClient.Do(req)
	if err != nil {
		return err
	}
	defer res.Body.Close()
	body, err := io.ReadAll(res.Body)
	if err != nil {
		return err
	}
	fmt.Println(string(body))
	return nil
}
